apps/cookbook/src/app/showcase/calendar-showcase/calendar-showcase.component.html

```diff
@@ -1,78 +1,118 @@
-<div class="example">
-  <h2>Default (transparent background)</h2>
-  <div class="calendar-example">
-    <cookbook-calendar-example></cookbook-calendar-example>
-    <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
+<p>
+  The calendar is a picker interface that displays a date grid with month navigation, for users to
```

Comment:
Consider reusing the description from the component overview: 
![image](https://github.com/user-attachments/assets/a3cd39de-5f62-470e-94b8-93010cd08ffd)

Reply:
Good catch, we need to change the description in the overview as well. 

I changed it in the calendar showcase because the existing one is hard to understand, in agreement with @henrikvoetmand